cmd/monitor.go

```diff
@@ -0,0 +1,245 @@
+			for _, params := range filesParams {
+				ch <- params
+			}
+			close(ch)
+		}()
+
+	case mode.IsRegular():
+		// Upload only one file to remote
+		go func() {
+			params := uploadParams{localPath, remotePath}
+			ch <- params
+			close(ch)
+		}()
+	default:
+		return errors.New("Not a regular file or folder")
+	}
+
+	client, err := NewAPIClient()
+	if err != nil {
+		return err
+	}
+
+	s := &monitorFileUpload{uploader: client.NewMonitorUploader()}
+	c := utils.NewCoordinator(viper.GetInt("threads"))
+	c.DoWithItemsFromChannel(s, ch)
+	return nil
+}
+
+var monitorItemUploadCmdHelp = `Upload a file or files contained in a folder.
+
+This command receives one file or folder path and uploads them to your
+VirusTotal Monitor account. It returns uploaded the file paths followed by their
+corresponding monitor ID.
+You can use the "vt monitor items [monitor_id]" command for retrieving
+information about the it.`
+
+var monitorItemUploadCmdExample = `  vt monitor item upload foo.exe /remote_folder/foo.exe
+  vt monitor item upload myfolder/ /another_remote_folder/`
+
+// NewMonitorItemsUploadCmd returns a new instance of the 'mointor upload file' command.
+func NewMonitorItemsUploadCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "upload [file/folder] [remote_path]",
+		Short:   "Upload one or more files to your account",
+		Long:    monitorItemUploadCmdHelp,
+		Example: monitorItemUploadCmdExample,
+		Args:    cobra.ExactArgs(2),
+		RunE:    runMonitorItemUpload,
+	}
+	addThreadsFlag(cmd.Flags())
+	return cmd
+}
+
+var monitorItemsCmdExample = `  vt monitor items list
+  vt monitor items list --filter "path:/myfolder/" --include path
+  vt monitor items list --filter "tag:detected" --include path,last_analysis_results.*.result,last_detections_count`
+
+// NewMonitorItemsCmd returns a new instance of the 'monitor_item' command.
+func NewMonitorItemsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "items [monitor_id] [flags]",
+		Short:   "Manage monitor items",
+		Example: monitorItemsCmdExample,
+		Args:    cobra.MinimumNArgs(1),
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			re, _ := regexp.Compile(base64RegExp)
+			p, err := NewPrinter(cmd)
+			if err != nil {
+				return err
+			}
+			return p.GetAndPrintObjects("monitor/items", args, re)
+		},
+	}
+
+	addThreadsFlag(cmd.Flags())
+
+	cmd.AddCommand(NewMonitorItemsListCmd())
+	cmd.AddCommand(NewMonitorItemsUploadCmd())
+
+	addRelationshipCmds(cmd, "monitor/items", "monitor_item", "[monitor_id]")
+
+	return cmd
+}
+
+var monitorCmdHelp = `Manage your VirusTotal Monitor account.
+
+This command allows you to manage the contents of your account and retrieve
+information about analyses performed to your collection.
+
+Reference:
+  https://developers.virustotal.com/v3.0/reference#monitor`
+
+// NewMonitorCmd returns a new instance of the 'monitor' command.
+func NewMonitorCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "monitor [subcommand]",
```

Comment:
The `[subcommand]` part is not necessary, in fact I've noticed that `NewHuntingCmd` does the same thing, but I'm going to fix it.

Reply:
done 92f8a89ddf28f4295f403dc802cb18254fecc136